frontend/model/contracts/chatroom.js

```diff
@@ -224,10 +225,7 @@ sbp('chelonia/defineContract', {
       },
       sideEffect ({ contractID, hash, meta }) {
         emitMessageEvent({ contractID, hash })
-
-        if (sbp('chelonia/contract/isSyncing', contractID)) {
-          updateUnreadPosition({ contractID, hash, createdDate: meta.createdDate })
-        }
+        updateUnreadPosition({ contractID, hash, createdDate: meta.createdDate })
```

Comment:
So the logic here completely reversed. Before, if it was called from a contract sync, we would update the unread position. Now, it no longer does.

Is this change intentional?

(Same question for the line above on line 208)

Reply:
Nice catch. Actually, I didn't notice that it reverses the previous workflow.
But the problem is that I can't understand the previous workflow to set `UnreadPosition` while the chatroom contract is syncing.
It makes sense to set `UnreadPosition` after joining the chatroom. So no need to set `UnreadPosition` while syncing the contract and the current solution is correct.